x-pack/plugins/monitoring/server/lib/create_query.js

```diff
@@ -26,6 +26,14 @@ export const createTypeFilter = (type) => {
   };
 };
 
+function getUuidFieldFromType(type) {
```

Comment:
We need to either implicitly figure this out or explicitly require it for each metric. It's an issue [once this is fixed](https://github.com/elastic/kibana/issues/23712)